typings/index.ts

```diff
@@ -380,6 +382,26 @@ client.on('message', ({ channel }) => {
   channel.send({ another: 'property' });
 });
 
+client.on('interaction', async interaction => {
+  if (!interaction.isCommand()) return;
+
+  const row = new MessageActionRow();
+
+  //@ts-expect-error
+  const badRow = new MessageActionRow({});
```

Comment:
Also move invalid ones to the bottom so we test valid code first, and invalid ones later.
```suggestion
  // @ts-expect-error
  void new MessageActionRow({});
```

The `void` here is to not create an unused variable, while also not making a useless expression.